recipe/meta.yaml

```diff
@@ -26,16 +32,14 @@ requirements:
     - matplotlib
 
 test:
-  source_files:
-    - bin/igorbinarywave.py
-    - bin/igorpackedexperiment.py
   imports:
     - igor.binarywave
     - igor.packed
     - igor.igorpy
+    - bin
```

Comment:
This will run the python command `import bin`. This happens to work, but isn't something you want here. :)

Reply:
thanks for stepping in @dougalsutherland!
Sorry for my ignorance, but why do you say it's something I don't want here? Because `bin` is not meant to be imported as a module?

Reply:
No, I mean that it doesn't cause any problems, but it also doesn't help anything – and it only happens to work. It's not like the `igor` package installs a Python package named `bin`; the tests just happen to be run from a directory that contains a `bin` directory. You should just delete it.

Reply:
ok yes, I see your point, thanks for clarifying!